assets/dev/scss/editor/panel/controls/_color.scss

```diff
@@ -21,3 +21,9 @@
 		}
 	}
 }
+
+input.pcr-clear {
```

Comment:
What is `pcr`? can you make it more explained?

Reply:
yes I can add a comment. pcr is a prefix that belongs to the color picker lib